lib/node-utils/index.ts

```diff
@@ -525,21 +525,27 @@ const matcherNamesHandlePromise = [
 	'rejects',
 	'toResolve',
 	'toReject',
+	'toBeRejected',
+	'toBeRejectedWith',
+	'toBeRejectedWithError',
+	'toBePending',
+	'toBeResolved',
+	'toBeResolvedTo',
 ];
 
 /**
- * Determines whether a node belongs to an async assertion
- * fulfilled by `resolves` or `rejects` properties or
- *  by `toResolve` or `toReject` jest-extended matchers
- *
+ * Determines whether a node belongs to an async assertion that is fulfilled by:
+ * - `resolves` or `rejects` properties
+ * - `toResolve` or `toReject` jest-extended matchers
+ * - jasmine async matchers
  */
-export function hasClosestExpectResolvesRejects(node: TSESTree.Node): boolean {
+export function hasClosestExpectHandlesPromise(node: TSESTree.Node): boolean {
```

Comment:
This util is used by `isPromiseHandled`, which is used by these rules:
- `await-async-queries`
- `await-async-utils`
- `await-async-events`

You added valid tests for the first two, can you add similar tests for the third one?

Reply:
added in db05d13d673bfa2dd4f5d175c42c3e3f092e2040. i noticed there weren't any `await-async-events` tests for the other async matchers either, so i added those as well.